src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -119,6 +127,23 @@ module sonic-portchannel {
 
 			description "PORTCHANNEL_INTERFACE part of config_db.json";
 
+			list PORTCHANNEL_INTERFACE_LIST {
+                                key "name";
+
+                                leaf name {
```

Comment:
This file is using tabs, we can keep the same format. Later in a single shot, we can convert from TAB --> SPACEs.

Reply:
In the next PR, in a single shot, we can convert from TAB --> SPACEs.

Reply:
@ArthiSivanantham : Shall we fix the TAB spaces on sonic-portchannel.yang now itself using these three steps? It will change quickly in a single shot. This will replace all the tabs with "4 spaces". I just tried this in sonic-portchannel.yang. Can you please try this and push the changes to this PR itself? Thanks for addressing all other comments.
```
* open sonic-portchannel.yang in vim
* :set list
* :set tabstop=4
* :set expandtab
* :retab
```

Reply:
Addressed. I've converted all tabs to spaces.